measures/measures-schema.yaml

```diff
@@ -197,3 +205,55 @@ definitions:
       - thoracicSurgery
       - urology
       - vascularSurgery
+
+  eligibilityOption:
+    type: object
+    properties:
+      submissionMethods: { $ref: #/definitions/qualityCodesSubmissionMethods }
+      sexCode:
+        enum: [M, F]
+      minAge:
+        type: integer
+      maxAge:
+        type: integer
+      diagnosisCodes: { $ref: #/definitions/arrayOfStringIdentifiers }
+      additionalDiagnosisCodes: { $ref: #/definitions/arrayOfStringIdentifiers }
+      procedureCodes:
+        type: array
+        items: { $ref: #/definitions/codeObject }
+    required: [submissionMethods]
+
+  performanceOption:
+    type: object
+    properties:
+      submissionMethods: { $ref: #/definitions/qualityCodesSubmissionMethods }
+      optionType:
+        enum: [performanceMet, performanceNotMet, performanceExclusion]
```

Comment:
This will likely be changing as a head's up! Waiting for final approval but performanceExclusion will likely be divided into eligablePopulationExclusion and eligiblePopulationException (field names TBD).